packages/web-app/src/components/appli/Entry/index.jsx

```diff
@@ -30,16 +32,10 @@ import { EntranceForm } from '../EntitiesForm';
 import SensitiveCaveWarning from './SensitiveCaveWarning';
 import AuthorLink from '../../common/AuthorLink';
 
-const EntryProperties = ({ isSensitive }) => (
-  <>
-    {isSensitive && <SensitiveCaveWarning />}
-    <EntryMap />
-    <Properties />
-  </>
-);
-EntryProperties.propTypes = {
-  isSensitive: PropTypes.bool
-};
+const HalfSplitContainer = styled.div`
+  display: flex;
+  ${isMobile && 'flex-direction: column;'}
```

Comment:
I disagree with this. 
Responsive design is not about the device: it's about the device screen size. `isMobile` should be used only when we really want to know if the device has (or has not) some functionalities related to a desktop computer, tablet or mobile.

I know there are some cases in the code base using isMobile() wrongly, we should remove them too.

Here is an example of how to do this: 
```js
const AppBar = styled(MuiAppBar)(({ theme }) => ({
  backgroundColor: red;

  [theme.breakpoints.up("sm")]: {
    backgroundColor: orange,
  },
  [theme.breakpoints.up("md")]: {
    backgroundColor: yellow,
    color: black,
  },
  [theme.breakpoints.up("lg")]: {
    backgroundColor: green,
    color: white,
  },
}));
```

Reply:
Yes completely agree, the code has been changed